benchmark/src/main/java/org/eclipse/rdf4j/benchmark/FastRdfsForwardChainingSailBenchmark.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+
+package org.eclipse.rdf4j.benchmark;
+
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.sail.inferencer.fc.FastRdfsForwardChainingSail;
+import org.eclipse.rdf4j.sail.memory.MemoryStore;
+
+/**
+ * @author Håvard Mikkelsen Ottestad
+ */
+public class FastRdfsForwardChainingSailBenchmark extends RdfsBenchmark {
+
+
```

Comment:
And don't worry _too_ much about it. We appreciate your best effort to keep things clean, but if code formatting if "off" in a few places, it's not the end of the world.